shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -185,14 +203,36 @@ public Intent build(@NonNull Context context) {
     }
   }
 
+  // Used to cover the Activity until the 1st frame is rendered so as to
```

Comment:
Assuming this is all going away after merging with pr36

Reply:
Yeah, it will go away to the extent that it will be replaced with the solution that's in pr36